syntax = "proto3";
import "google/protobuf/empty.proto";
import "google/protobuf/timestamp.proto";

message RawMeteoData {
  string id = 1;
  float temperature = 2;
  float humidity = 3;
  google.protobuf.Timestamp timestamp = 4;
}

message RawPollutionData {
  string id = 1;
  float co2 = 2;
  google.protobuf.Timestamp timestamp = 3;
}

service LBService {
  rpc send_meteo_data (RawMeteoData) returns (google.protobuf.Empty) {}
  rpc send_pollution_data (RawPollutionData) returns (google.protobuf.Empty) {}
}